static/js/components/StarList.jsx

```diff
@@ -19,14 +19,16 @@ const StarListBody = ({ starList, facility, setFacility, setStarList }) => {
     <div className={styles.starListDiv}>
       <code className={styles.starList}>
         <div className={styles.codeText}>
-          {starList &&
-            starList.map((item, idx) => (
-              // eslint-disable-next-line react/no-array-index-key
-              <React.Fragment key={idx}>
-                {item.str}
-                <br />
-              </React.Fragment>
-            ))}
+          <pre>
```

Comment:
we need a <pre> here to insure that the spaces are properly rendered. Otherwise, even with &nbsp; those spaces are not preserved in showing the starlist.